src/compounds/product-table/src/components/rowWrapper.tsx

```diff
@@ -22,12 +22,22 @@ const checkClickTargetIsAccordion = (n: number, e: any) => {
   return false
 }
 
+const checkClickTargetIsLink = (e: any) => {
+  if (e.target.nodeName.toLowerCase() === 'a') {
+    return true
+  }
+}
+
 const linkWrapper = (
   link: string,
   children: React.ReactNode,
   headerImage?: React.ReactNode
 ) => {
   const handleClick = (e: any) => {
+    if (checkClickTargetIsLink(e)) {
```

Comment:
We could merge the two ifs: if(checkClickTargetIsAccordion(10, e) && !checkClickTargetIsLink) and then return e by default